ethereum/core/src/testFixtures/java/tech/pegasys/teku/core/BlockProposalTestUtil.java

```diff
@@ -123,8 +112,9 @@ public SignedBlockAndState createBlock(
         exits.orElse(blockBodyLists.createVoluntaryExits()));
   }
 
-  private static Eth1Data get_eth1_data_stub(BeaconState state, UInt64 current_epoch) {
-    UInt64 epochs_per_period = UInt64.valueOf(EPOCHS_PER_ETH1_VOTING_PERIOD);
+  private Eth1Data get_eth1_data_stub(BeaconState state, UInt64 current_epoch) {
+    final SpecConfig specConfig = spec.atSlot(state.getSlot()).getConfig();
+    UInt64 epochs_per_period = UInt64.valueOf(specConfig.getEpochsPerEth1VotingPeriod());
     UInt64 voting_period = current_epoch.dividedBy(epochs_per_period);
```

Comment:
nit: I think we can skip creating a `UInt64` of `epochs_per_period` here since `dividedBy` can take a `long` directly.